Dockerfile_openvino_base

```diff
@@ -0,0 +1,22 @@
+FROM openvino/ubuntu18_runtime
+USER root
+RUN apt-get update && apt-get install -y python3-dev
+RUN pip3 install virtualenv
+WORKDIR /ie-serving-py
+RUN chown openvino .
+USER openvino
+COPY requirements.txt /ie-serving-py/
+RUN virtualenv -p python3 .venv && \
+    . .venv/bin/activate && pip3 --no-cache-dir install -r requirements.txt
+COPY start_server.sh setup.py version /ie-serving-py/
+COPY ie_serving /ie-serving-py/ie_serving
+
+RUN . .venv/bin/activate && pip3 install .
+ENV PYTHONPATH=/opt/intel/openvino/python/python3.6:/opt/intel/openvino/python/python3
+ARG OV_PATH=/opt/intel/openvino/deployment_tools
+ENV LD_LIBRARY_PATH=\
+$OV_PATH/ngraph/lib:\
+$OV_PATH/inference_engine/external/gna/lib:\
+$OV_PATH/inference_engine/external/mkltiny_lnx/lib:\
+$OV_PATH/inference_engine/external/tbb/lib:\
+$OV_PATH/inference_engine/lib/intel64
```

Comment:
This should be formatted to human-readable indents or use += 
Also end with newline.